west.yml

```diff
@@ -64,11 +64,7 @@ manifest:
     - name: hal_espressif
       revision: f525ee6f468c2c68119731e1453ad72d189a1277
       path: modules/hal/espressif
-      groups:
-        - hal
```

Comment:
why the removal here of `hal` group?

Reply:
In fact, I just fixed it because some previous PR messed with it. It has duplicated entries:

```c
    - name: hal_espressif
      revision: f525ee6f468c2c68119731e1453ad72d189a1277
      path: modules/hal/espressif
      groups:
        - hal
      west-commands: west/west-commands.yml
      revision: c21d252a2188fcb60d52dceb6b018a9f4a371817
      path: modules/hal/espressif
      groups:
        - hal
```